src/main/java/com/github/kokorin/jaffree/process/BaseStdReader.java

```diff
@@ -86,6 +90,10 @@ public T read(final InputStream stdOut) {
                         LOGGER.error(logMessage.message);
                         break;
                 }
+
+                if (logMessage.logLevel.isErrorOrHigher()) {
+                    errorLogMessages.add(0, logMessage);
```

Comment:
it's better to move message collection next to logging errors at line 90: 
```java
LOGGER.error(logMessage.message);
errorLogMessages.add(logMessage);
```

Reply:
Cool, I've fixed the error collection and changed the behaviour to LIFO.

For me it's the last error I'm usually interested in the last error; that's why I initially decided to add the errors in reverse order and use a LinkedList (to avoid ArrayList reallocations when adding to the front). Thinking about this again it's probably better just to keep them in chronological order.